fabtests/Makefile.am

```diff
@@ -454,6 +455,14 @@ regression_sighandler_test_LDADD = libfabtests.la
 regression_sighandler_test_CFLAGS = \
 	$(AM_CFLAGS)
 
+common_check_hmem_SOURCES = \
+	common/check_hmem.c
```

Comment:
can we directly use the `fi_info` bin from libfabric?

Reply:
we could, but I wasn't sure if there was a way to guarantee it was the libfabric that fabtests was built against. There's probably a way to do so I can try looking into it

Reply:
I'd guess we can use the configure --prefix info, I'm not sure though if we can pass the configure/prefix info to python

Reply:
Yeah, I think it is not straightforward to use it. Do not bother then.